Building DAG of jobs...
Provided cores: 1
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	merge
	1	plot
	2

[Wed Feb 20 17:32:47 2019]
rule merge:
    input: src/data-management/merge.R, src/data/Players.csv, src/data/Seasons_Stats.csv
    output: out/data/data_merged.csv
    log: logs/merge.Rout
    jobid: 1

[Wed Feb 20 17:32:50 2019]
Finished job 1.
1 of 2 steps (50%) done

[Wed Feb 20 17:32:50 2019]
rule plot:
    input: src/graphs/graph.R, out/data/data_merged.csv
    output: out\graphs\graph.pdf
    log: logs/plot.Rout
    jobid: 0

[Wed Feb 20 17:32:55 2019]
Error in rule plot:
    jobid: 0
    output: out\graphs\graph.pdf
    log: logs/plot.Rout

RuleException:
CalledProcessError in line 29 of C:\Users\ksrinivasan\OneDrive\Zurich\Courses\others\programming_winter\PP4RS_assignment_KG\Snakefile:
Command 'Rscript src/graphs/graph.R        --data out/data/data_merged.csv        --out out\graphs\graph.pdf > logs/plot.Rout 2>&1' returned non-zero exit status 1.
  File "C:\Users\ksrinivasan\OneDrive\Zurich\Courses\others\programming_winter\PP4RS_assignment_KG\Snakefile", line 29, in __rule_plot
  File "c:\users\ksrinivasan\appdata\local\continuum\anaconda3\lib\concurrent\futures\thread.py", line 57, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: C:\Users\ksrinivasan\OneDrive\Zurich\Courses\others\programming_winter\PP4RS_assignment_KG\.snakemake\log\2019-02-20T173247.549930.snakemake.log
